src/functionalTest/resources/features/F-010/S-106_DeleteDataForRoleAssignments.td.json

```diff
@@ -0,0 +1,38 @@
+{
+  "_guid_": "S-106_DeleteDataForRoleAssignments",
+  "productName": "Role Assignment Service",
+  "operationName": "Delete Role Assignments by Process",
+  "method": "DELETE",
+  "uri": "{{ROLE_ASSIGNMENT_URL}}/am/role-assignments?process={process}&reference={reference}",
+  "specs": [
+    "to delete role assignments just created above",
+    "to delete existing role assignments corresponding to the test actorId"
+  ],
+  "users": {
+    "invokingUser": {
+      "_extends_": "AmBeftaUser1"
+    }
+  },
+  "request": {
+    "headers": {
+      "Authorization": "[[DEFAULT_AUTO_VALUE]]",
+      "ServiceAuthorization": "[[DEFAULT_AUTO_VALUE]]",
+      "X-Correlation-ID": "003352d0-e699-48bc-b6f5-5810411e60af"
+    },
+    "pathVariables": {
+      "process": "staff-organisational-role-mapping",
```

Comment:
The true process name for ORM Judicial is `judicial-organisational-role-mapping`

Reply:
i.e. looking a history table in RAS AAT these roles were creted once on the 08-Dec and then have never been deleted or updated since (i.e. they remain unchnged).

if you think the same may be true for other existing JUDICIAL ORM FTAs then we probably should look to update at some point in future.

Reply:
Updated.

Reply:
Resolved in 79569ab9c9b1085fc204051868a90b5e8bc60293.